src/pages/news/news-details/news-details.js

```diff
@@ -65,226 +89,154 @@ const NewsDetails = ({ match }) => {
+    preferredLanguages.length > 0
+      ? preferredLanguages.map((lang, index) => (
+        <Tab label={lang} key={index} />
+      ))
+      : null;
   if (loading) {
     return <LoadingBar />;
   }
 
   return (
     <div className={classes.detailsContainer}>
-      <form className={classes.form} onSubmit={newsSaveHandler}>
-        <FormControl className={classes.newsDetails}>
-          <Grid container spacing={1}>
-            <Grid item xs={12}>
-              <Paper className={classes.newsItemUpdate}>
-                <TextField
-                  id='authorPhoto'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Фото автора'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={authorPhoto}
-                  onChange={(e) => setAuthorPhoto(e.target.value)}
-                  required
-                />
-                <TextField
-                  id='newsImage'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Головне зображення'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={newsImage}
-                  onChange={(e) => setNewsImage(e.target.value)}
-                  required
-                />
-              </Paper>
-            </Grid>
-
-            <Grid item xs={6}>
-              <Paper className={classes.newsItemUpdate}>
-                <TextField
-                  id='ukAuthorName'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Автор (укр.)'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={ukAuthorName}
-                  onChange={(e) => ukSetAuthor(e.target.value)}
-                  required
-                />
-                <TextField
-                  id='ukTitle'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Заголовок (укр.)'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={ukTitle}
-                  onChange={(e) => ukSetTitle(e.target.value)}
-                  required
+      {newsArticle !== null ? (
+        <Formik
```

Comment:
Use `useFormik` instead of the `renderProps` component. Look at the login form implementation

Reply:
I can't replace it due to the features of Formik initial state.